### DISCUSSION TITLE
在哪里可以找到支持Axure的最新版本资源（组件库）？

### DISCUSSION THREAD

**Discussion by huastonecode** [ant-design/ant-design#39797] [General]
### Reproduction link

[https://no.websit](https://no.websit)

### Steps to reproduce

no

### What is expected?

请明确目前对Axure组件库的支持情况

### What is actually happening?

目前在官网差不到相关信息

| Environment | Info |
| --- | --- |
| antd | 5.1.0 |
| React | latest |
| System | andiod |
| Browser | latest |

**Comment by afc163** [comment]
我们不再对 Axure 设计资源提供支持。

**Comment by huastonecode** [comment]
请教，Ant Design有替代产品推荐吗？ 谢谢。
------------------------------------------------------------------
发件人：afc163 ***@***.***>
发送时间：2022年12月25日(星期日) 18:37
收件人：ant-design/ant-design ***@***.***>
抄　送：huastonecode ***@***.***>; Author ***@***.***>
主　题：Re: [ant-design/ant-design] 在哪里可以找到支持Axure的最新版本资源（组件库）？ (Discussion #39797)
我们不再对 Axure 设计资源提供支持。

**Comment by sweepingmonk123** [comment]
为啥不再提供axure的支持了呀

**Reply by arvinxx** [reply]
因为我们内部不再使用 Axure 了，所以不会提供 Axure 支持。就像 Figma 资源、MasterGo 资源等等各类设计资产一样我们不支持。鉴于 ROI 考虑，我们不可能为每一个平台都制作相应的设计资产的，请谅解。 @767746649zyy  @geekidentity

**Reply by 767746649zyy** [reply]
没人用是啥意思 没人在axure里使用antd资源了吗 我们领导还要让我把修改样式后的antd资源加载到axure里呢

**Reply by geekidentity** [reply]
为啥不支持了呢

**Comment by pkaq** [comment]
不支持那4.0的旧版起码留个连接把
